packages/Sortable/src/Sortable.js

```diff
@@ -0,0 +1,70 @@
+import React from "react";
+import PropTypes from "prop-types";
+import uuidv4 from "uuid/v4";
+import { DragDropContext, Droppable } from "react-beautiful-dnd";
+import SortableContext from "./SortableContext";
+import Item from "./components/Item/Item";
```

Comment:
Should the `Item` component live under `Sortable`? It wouldnt be used without it.

Reply:
What do you mean by "live under"?  It's not a component the app consumer will need to know about, so I didn't bother exposing it as `<Sortable.Item>`.

Reply:
```
import Item from "./Sortable/Item/Item";
```

Reply:
oh nevermind, its good